website/docs/developer/development-guide/documentation/documentation-add-content.md

```diff
@@ -0,0 +1,78 @@
+---
+id: contrib-documentation-add
+title: Adding New Content
+desc: Provides instruction on adding a page to ORCA documentaion.
+---
+
+## Add a New Documentation Page
+
+Adding a new page should be as simple as writing some documentation in markdown,
```

Comment:
```suggestion
Creating a new page should be as simple as writing some documentation in markdown,
```

Because "adding" makes me think that you're describing the whole process of adding a page (including json updates)